File: week-02/day-01/29_draw_pyramid.py
# Write a program that reads a number from the standard input, then draws a
# pyramid like this:
#
#
#    *
#   ***
#  *****
# *******
#
# The pyramid should have as many lines as the number was

num = input("Enter a number: ")
num = int(num)
str = "*"

for i in range(num):
    print(num * " " + str)
    str += "*" * 2
    num -= 1

File: week-07/day-01/29_draw_pyramid.js
'use strict';

let lineCount = 4;
let star = '*';
let space = ' ';

// Write a program that draws a
// pyramid like this:
//
//
//    *
//   ***
//  *****
// *******
//
// The pyramid should have as many lines as lineCount is

for (let i = lineCount; i > 0; i--) {
    console.log(space.repeat(i) + star);
    star += '**';
}
